### ISSUE TITLE
Tags disappear when network drive ist disconnected

### ISSUE THREAD

**Issue by StachLeee** [files-community/Files#13502]
### Description

When a network drive is can't be found (because of an network disconnect for example), on the startpage all tagged files and folders located on this drive disappear. Even if the drive can be found again, the tagged items remain gone. You have to open each file path individually to make the items reappear.

The behaviour should be such that the items are still displayed. Either an error message appears when trying to open them or it is graphically displayed that they are not available or they should should appear as soon as the network drive is available again.

### Steps To Reproduce

Tags-Widget on the Homescreen needs to be activated.
1. Connect a Network drive
2. tag some files or folders on this drive
3. Make it so that the network drive cannot be found (for example disconnect the PC from network)
4. The network drive can't be found. After the corresponding error message shows up all tagged files and folder disappear from the startpage.
5. Reconnect the PC to the network --> network drive is available again but tagged files and folders remain gone on the startpage

### Requirements

- tagged files and folders should remain displayed when the network drive cannot be found 
OR
- tagged files and folders should appear as soon as the network drive is available again

### Files Version

2.5.21.0

### Windows Version

10.0.19044.3448

### Log File

[debug.log](https://github.com/files-community/Files/files/12865723/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, Having the tags displayed when the drive is disconnected isn't possible as Files doesn't know there tags there as it can't search it. 
As for searching for tags when the drive connects is possible, it would also take a lot of resources searching the entire drive.

**Comment by yaira2** [comment]
> Reconnect the PC to the network --> network drive is available again but tagged files and folders remain gone on the startpage

Does opening a new tab display the tags?

**Comment by StachLeee** [comment]
> Does opening a new tab display the tags?

No, that does not help. Neither does closing and opening the app.

**Comment by StachLeee** [comment]
> As for searching for tags when the drive connects is possible, it would also take a lot of resources searching the entire drive.

Could Files simply save "dummies" of the tagged files and folders (with their path) from the last session without searching for them directly at startup?

**Comment by yaira2** [comment]
I meant, does opening a new tab help after reconnecting the drive?

**Comment by StachLeee** [comment]
No, just opening a new tab does not help (after reconnection). You need to 
1) open each folder that contains tagged objects (also after reconnection) and after that 
2) open the startpage again / open new tab and the objects will reappear

**Comment by yaira2** [comment]
Thanks for the clarification, this is indeed a bug. The expected behavior is for tags widget to update when a drive is connected/disconnected.

**Comment by yaira2** [comment]
We're going to merge this with #6182 where we will continue to track this bug.
